### PR TITLE
Add forensic dashboard realated queries in all savedsearches

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -819,6 +819,7 @@ search = `ms_obj_changes_base_cat_customised("groupmembershipchanged")`\
 | `cs_human_readable_time_format(_time, event_time)` \
 | `cs_ad_group_membership_changed_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `ms_obj_changes_base_cat_customised("groupmembershipchanged")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:822]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -920,6 +921,7 @@ search = `cs_wineventlog_security` (EventCode=4740) \
 | `cs_human_readable_time_format(_time, event_time)` \
 | `cs_ad_user_locked_out_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_wineventlog_security` (EventCode=4740)
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:924]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -949,6 +951,7 @@ search = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) \
 | table _time, ComputerName, Target_Account_Name, Target_Account_Domain, Subject_Account_Name Subject_Account_Domain Caller_Workstation, msad_action, action, EventCode \
 | `cs_ad_password_change_outside_working_hour_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) | eval time = _time | `cs_generate_time_fields_for_filter(time)` | where time_wday="Saturday" OR time_wday="Sunday" OR time_hour<6 OR time_hour>19 | `cs_remove_time_fields_for_filter(time)`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:954]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 4]
```diff
@@ -979,6 +982,7 @@ search = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) \
 | `cs_ad_multiple_password_changes_filter` \
 | sort - count
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794)
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:985]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 5]
```diff
@@ -1009,6 +1013,7 @@ search = `cs_wineventlog_security` EventCode IN (4720, 4726) \
 | where total_actions>=`cs_ad_bulk_user_creation_deletion_count_limit` \
 | `cs_ad_bulk_user_creation_deletion_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_wineventlog_security` EventCode IN (4720, 4726)
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1016]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 6]
```diff
@@ -1244,6 +1249,10 @@ search = `cs_palo` sourcetype="pan:system" "Commit job succeeded" \
 | table _time serial_number dvc dvc_name description \
 | `cs_palo_commit_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_palo` sourcetype="pan:system" "Commit job succeeded"
+action.cyences_notable_event_action.system_compromised_search = | stats count by dvc_name
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1253]: | stats count by dvc, dvc_name

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 7]
```diff
@@ -1398,6 +1407,9 @@ search = `cs_o365` _index_earliest=-61m@m _index_latest=-1m@m (sourcetype="o365:
 | table LastUpdatedTime host Service Status \
 | sort host Status | `cs_o365_service_not_operational_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` (sourcetype="o365:service:healthIssue" OR (sourcetype="o365:service:status" source="*CurrentStatus")) | eval Status = coalesce(status, Status), Service = coalesce(service, WorkloadDisplayName)
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1410]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 8]
```diff
@@ -1431,6 +1443,7 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-25h@m
 | `cs_human_readable_time_format(_time, event_time)` \
 | table event_time, memberUpn | rename memberUpn as "External Users" | `cs_o365_external_users_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=MicrosoftTeams Operation=MemberAdded Members{}.UPN=*\#ext\#*
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1446]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 9]
```diff
@@ -1459,6 +1472,7 @@ search = `cs_o365_managementactivity_change("AuthorizationPolicy")` _index_earli
 | `cs_o365_authorizationpolicy_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("AuthorizationPolicy")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1475]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 10]
```diff
@@ -1487,6 +1501,7 @@ search = `cs_o365_managementactivity_change("Policy")` _index_earliest=-61m@m _i
 | `cs_o365_policy_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("Policy")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1504]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 11]
```diff
@@ -1515,6 +1530,7 @@ search = `cs_o365_managementactivity_change("Role")` _index_earliest=-61m@m _ind
 | `cs_o365_role_change_filter`   \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("Role")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1533]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 12]
```diff
@@ -1543,6 +1559,7 @@ search = `cs_o365_managementactivity_change("Group")` _index_earliest=-61m@m _in
 | `cs_o365_group_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("Group")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1562]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 13]
```diff
@@ -1571,6 +1588,7 @@ search = `cs_o365_managementactivity_change("GroupMembership")` _index_earliest=
 | `cs_o365_group_membership_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("GroupMembership")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1591]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 14]
```diff
@@ -1599,6 +1617,7 @@ search = `cs_o365_managementactivity_change("User")` _index_earliest=-61m@m _ind
 | `cs_o365_user_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("User")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1620]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 15]
```diff
@@ -1627,6 +1646,7 @@ search = `cs_o365_managementactivity_change("ServicePrincipal")` _index_earliest
 | `cs_o365_serviceprincipal_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("ServicePrincipal")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1649]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 16]
```diff
@@ -1655,6 +1675,7 @@ search = `cs_o365_managementactivity_change("Application")` _index_earliest=-61m
 | `cs_o365_application_change_filter`  \
 | table event_time, Actor, Command, Target_ObjectID, Target_DisplayName, ResultStatus, ModifiedProperties, MetaData
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_managementactivity_change("Application")`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1678]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 17]
```diff
@@ -1687,6 +1708,11 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m
 | `cs_human_readable_time_format(Last_Failed_Login)` \
 | `cs_o365_failed_login_due_to_mfs_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt")
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1711]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 18]
```diff
@@ -1721,6 +1747,11 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m
 | `cs_human_readable_time_format(Last_Failed_Login)` \
 | `cs_o365_failed_login_due_to_mfs_outside_country_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt")
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1750]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 19]
```diff
@@ -1749,6 +1780,7 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m
 | `cs_human_readable_time_format(Last_Success_Login)` \
 | `cs_o365_login_by_unknown_userid_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoggedIn UserId=Unknown
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1783]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 20]
```diff
@@ -1782,6 +1814,9 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m
 | `cs_human_readable_time_format(Last_Success_Login)` \
 | `cs_o365_success_login_outside_country_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity"  Workload=AzureActiveDirectory Operation=UserLoggedIn NOT LogonError=*
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1817]: Need to apply filter of outside home country.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 21]
```diff
@@ -1782,6 +1814,9 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m
...
+action.cyences_notable_event_action.system_compromised_search = | stats count by user
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365` sourcetype="o365:management:activity"  Workload=AzureActiveDirectory Operation=UserLoggedIn NOT LogonError=* user="$row.user$"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1819]: Need to apply filter of outside home country.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 22]
```diff
@@ -1814,6 +1849,9 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m
 | `cs_human_readable_time_format(Last_Failed_Login)` \
 | `cs_o365_failed_login_due_to_conditional_access_policy_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity"  Workload=AzureActiveDirectory Operation=UserLoginFailed LogonError="BlockedByConditionalAccess"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1852]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 23]
```diff
@@ -1846,6 +1884,11 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-25h@m
 | `cs_human_readable_time_format(Last_Failed_Login)` \
 | `cs_o365_daily_login_failure_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1887]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 24]
```diff
@@ -1873,6 +1916,9 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m
 | table _time Name Severity AlertType Category f3u lon wl ad \
 | `cs_o365_security_compliance_alert_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="o365:management:activity" Workload=SecurityComplianceCenter Operation=AlertTriggered
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1919]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 25]
```diff
@@ -1937,6 +1983,11 @@ search = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" _index_earliest=-
 | sort - count \
 | `cs_email_daily_spam_email_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" Status=FilteredAsSpam 
+action.cyences_notable_event_action.system_compromised_search = | stats count by SenderAddress
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:1987]: The Attacker and Compromised system need to be reversed. (Debatable, we'll discuss on call further.)

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 26]
```diff
@@ -2333,6 +2384,11 @@ search = `cs_o365_defender_atp_alerts`\
 | table computerDnsName, count, firstTime, lastTime, lastUpdateTime, resolvedTime, incidentId, title, threatFamilyName, threatName, category, classification, severity, status, investigationId, invenstigationState, assignedTo, detectionSource, detectorId, id, machineId\
 | `cs_o365_defender_atp_alerts_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_defender_atp_alerts`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:2387]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 27]
```diff
@@ -2950,6 +3006,11 @@ search = | tstats `cs_summariesonly_authentication` count, max(_time) as Last_Su
 | `cs_human_readable_time_format(Last_Success_Login)` \
 | `cs_authentication_successful_vpn_login_outside_home_country_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = index=* `cs_vpn_indexes` tag=authentication action="success" dest_category="vpn_auth"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3009]: Outside home country filter needs to be added.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 28]
```diff
@@ -2950,6 +3006,11 @@ search = | tstats `cs_summariesonly_authentication` count, max(_time) as Last_Su
...
+action.cyences_notable_event_action.system_compromised_search = | stats count by Dest
+action.cyences_notable_event_action.system_compromised_drilldown = index=* `cs_vpn_indexes` dest="$row.Dest$" tag=authentication action="success" dest_category="vpn_auth"
+action.cyences_notable_event_action.attacker_search = | stats count by user
+action.cyences_notable_event_action.attacker_drilldown = index=* `cs_vpn_indexes` user="$row.user$" tag=authentication action="success" dest_category="vpn_auth"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3013]: Outside home country filter needs to be added.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 29]
```diff
@@ -3041,6 +3107,11 @@ search = | tstats `cs_summariesonly_authentication` count from datamodel=Authent
 | table _time user dest src app count City Region Country pct \
 | `cs_authentication_successful_vpn_login_from_new_location_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = index=* `cs_vpn_indexes` tag=authentication action="success" dest_category="vpn_auth"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3110]: Let's again discuss this, just in case if we can add filter here.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:3110]: **[Important]**
Please also create 1 Jira ticket to show only alerts on the **Overview page** which has "**action.cyences_notable_event_action = 1**".
Currently it's being determined by a macro, which is not properly being maintained and it's not a good idea anymore.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 30]
```diff
@@ -3073,6 +3144,11 @@ search = `cs_vpn_indexes` eventtype="cs_*_vpn_logout" duration>=`cs_authenticati
 | rename dest as Dest, user as User, duration as Duration \
 | `cs_authentication_vpn_long_running_session_disconnected_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_vpn_indexes` eventtype="cs_*_vpn_logout" duration>=`cs_authentication_vpn_session_duration_limit`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3147]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 31]
```diff
@@ -3155,6 +3231,11 @@ search = `cs_linux_users_with_previledge` \
 | `cs_human_readable_time_format(sudo_last_modified)` \
 | `cs_human_readable_time_format(user_last_modified)`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_linux_sudousers` | rex field=_raw "sudouser=(?<USERNAME>.*)" max_match=0
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3234]: ok

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 32]
```diff
@@ -3205,9 +3286,12 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-I-CONNECT" \
 | rex "CONNECT:\s(?<message>.*)" \
-| table _time message \
+| table _time dvc message \
 | `cs_cisco_ios_new_connection_for_user_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-I-CONNECT"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3292]: Please check to see if we have user or source_ip information to be used as a attacker.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 33]
```diff
@@ -3232,9 +3316,12 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-W-REJECT" \
 | rex "REJECT:\s(?<message>.*)" \
-| table _time message \
+| table _time dvc message \
 | `cs_cisco_ios_device_failed_login_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-W-REJECT"
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3322]: Please check to see if we have user or source_ip information to be used as a attacker.

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 29 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:3110]: > **[Important]** Please also create 1 Jira ticket to show only alerts on the **Overview page** which has "**action.cyences_notable_event_action = 1**". Currently it's being determined by a macro, which is not properly being maintained and it's not a good idea anymore.

Updated existing CY-414 Jira

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 32 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:3292]: Currently, the src_ip and user_name fields are not getting extracted.

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 33 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:3322]: Currently, the src_ip and user_name fields are not getting extracted.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 34]
```diff
@@ -2333,6 +2384,11 @@ search = `cs_o365_defender_atp_alerts`\
 | table computerDnsName, count, firstTime, lastTime, lastUpdateTime, resolvedTime, incidentId, title, threatFamilyName, threatName, category, classification, severity, status, investigationId, invenstigationState, assignedTo, detectionSource, detectorId, id, machineId\
 | `cs_o365_defender_atp_alerts_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365_defender_atp_alerts`
+action.cyences_notable_event_action.system_compromised_search = | stats count by computerDnsName
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365_defender_atp_alerts` computerDnsName="$row.computerDnsName$"
+action.cyences_notable_event_action.attacker_search = | stats count by title
```
[the remark this answers is not in the document]
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:2390]: It's like a signature of attach. ie. "Outbound vertical port scan"

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 34 above)
[the remark this answers is not in the document]
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:2390]: As discussed in call. This is fine.

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 25 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:1987]: As discussed in call. This is fine.

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 32 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf:3292]: add src field in the attacker search as user is not present in all the events.

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 29 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/default/savedsearches.conf]: Updated search to limit the results

**Pull request merged**